docs/hooks.md

```diff
@@ -704,6 +704,18 @@ The number of elements in the given list.
         [hook{}("LIST.size")]
 ~~~
 
+## LIST.updateAll
+
+`LIST.updateAll(L1, index, L2)` creates a new list using a list `L2` of size `N`
+where the `N` elements of `L1` starting at `index` are replaced with the
+contents of `L2`. The result is `\bottom` if `index` is negative or part of `L2`
+would go of the end of `L1`.
```

Comment:
Could you reword this somehow? I'm having trouble understanding what `part of L2
would go of the end of L1` means.

Reply:
I used [this](https://runtimeverification.slack.com/archives/CC360GUTG/p1605892398253700?thread_ts=1605699058.240400&cid=CC360GUTG) explanation. I'll try to find a clearer one.